src/main/java/uk/ac/ebi/eva/pipeline/configuration/VariantJobsArgs.java

```diff
@@ -53,6 +56,12 @@
 @Component
 public class VariantJobsArgs {
     private static final Logger logger = LoggerFactory.getLogger(VariantJobsArgs.class);
+    private static final String DB_COLLECTIONS_FEATURES_NAME = "db.collections.features.name";
```

Comment:
private? I thought this was to avoid getting values from pipelineOptiones with hardcoded strings. Do we want these to be used only in the getters below? Not that I'm against it, I'm just curious